packages/amplify-cli-core/src/state-manager/stateManager.ts

```diff
@@ -76,6 +77,14 @@ export class StateManager {
     return this.getData<$TSTeamProviderInfo>(filePath, mergedOptions);
   };
 
+  getCustomPolicies = (categoryName: string, resourceName: string): CustomIAMPolicies | undefined => {
+    const filePath = pathManager.getCustomPoliciesPath(categoryName, resourceName);
+    if (!(fs.existsSync(filePath)) || !isCustomPoliciesFile(filePath)) {
+      return undefined;
+    }
```

Comment:
@luhanamz This is doing what JSONUtilities already does, you might not need it and then you can get rid of isCustomPoliciesFile(filepath)